README.md

```diff
@@ -38,23 +38,37 @@ code from GitHub.
 
 ## How to build and install ##
 
-To build with Autotools, see `docs/autotools.rst` for more information.
-(To build on GNU/Linux, Autotools is your choice.)
-To build on Windows, see `docs/windows.rst` for more information.
-To build on OSX, see `docs/osx.rst` for more information.
+To build with Autotools (Autoconf, Automake, and Libtool) on GNU/Linux, OSX, or Windows 10 WSL,
+```
+    $ git clone https://github.com/universal-ctags/ctags.git
+    $ cd ctags
+    $ ./autogen.sh
+    $ ./configure --prefix=/where/you/want # defaults to /usr/local
+    $ make
+    $ make install # may require extra privileges depending on where to install
+```
+
+See
+[`docs/autotools.rst`](https://github.com/universal-ctags/ctags/blob/master/docs/autotools.rst)
+for more information.
+
+To build on Windows, see
+[`docs/windows.rst`](https://github.com/universal-ctags/ctags/blob/master/docs/windows.rst)
+for more information.
+
+To build on OSX, see
+[`docs/osx.rst`](https://github.com/universal-ctags/ctags/blob/master/docs/osx.rst)
+for more information.
 
 ## Manual ##
-Man page (ctags.1) is generated only in Autotools based building process.
-In addition rst2man command is needed.
+Go to https://docs.ctags.io for the preformatted documentations.
```

Comment:
I'm afraid the reader uses a bit older version of u-ctags. How about the following sentence?

```
Go to https://docs.ctags.io for the preformatted documentation for the latest development version.
```

Reply:
Good suggestion.
It is fixed.